src/Emitter/SapiEmitterTrait.php

```diff
@@ -31,8 +31,9 @@ trait SapiEmitterTrait
      */
     private function assertNoPreviousOutput()
     {
-        if (headers_sent()) {
-            throw EmitterException::forHeadersSent();
+        $file = $line = null;
```

Comment:
Please define on separate lines

Reply:
I think what you need ... if not, please specify

Reply:
Unfortunately, unsuccessfully ... I assume that sending headers is not called in the SAP environment of tests, all my attempts at tests are unsuccessful ...